dist/ci/obs_testsuite_travis_install.sh

```diff
@@ -17,3 +19,7 @@ sudo apt-get update
 
 # Install the dependencies of the backend
 sudo apt-get install --force-yes travis-deps libxml-parser-perl libfile-sync-perl python-rpm python-urlgrabber python-sqlitecachec python-libxml2 createrepo libbssolv-perl sphinxsearch libjson-xs-perl libxml-simple-perl libgd-gd2-perl
+
+if [ "$TEST_SUITE" == "backend" ]; then
+  sudo apt-get install --force-yes libdevel-cover-perl
+fi
```

Comment:
If you're already conditionally install packages per test suite, why don't you skip the parts not needed for the backend test suite? :-)